### PR TITLE
{devel}[GCCcore/6.4.0] Doxygen v1.8.13

### PR DESCRIPTION
**Pull request by JackPerdue**
(created using `eb --new-pr`)
depends on PR ~~#4933~~

### CODE REVIEW

**Comment by boegel** [comment]
(close/open to retrigger Travis)

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2488.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/2417d286fa7388563c7bb7d70aed7fa8 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2087.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/794a9eda4b072d25f649358a0d66bea4 for a full test report.

File: easybuild/easyconfigs/d/Doxygen/Doxygen-1.8.13-GCCcore-6.4.0.eb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+
+source_urls = ['http://ftp.stack.nl/pub/users/dimitri/']
+sources = ['%(namelower)s-%(version)s.src.tar.gz']
+checksums = ['af667887bd7a87dc0dbf9ac8d86c96b552dfb8ca9c790ed1cbffaa6131573f6b']
+
+builddependencies = [
+    ('binutils', '2.28'),
+    ('Bison', '3.0.4'),
+    ('CMake', '3.8.2'),
```
Comment (boegel) [inline easybuild/easyconfigs/d/Doxygen/Doxygen-1.8.13-GCCcore-6.4.0.eb]: should be bumped to CMake 3.9.1, see #5073

**Comment by JackPerdue** [comment]
Again, I don't know why --update-pr pulled in the CMake.eb.... should be the same as the one already accepted

**Comment by boegel** [comment]
Same as in #4946, probably because you used `--robot` with `--update-pr`. Nothing to worry about if `CMake` is untouched, we can verify easily via `eb --review-pr`.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2403.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/d4517a4b7dfbec0815b7efac3b56080a for a full test report.

**Comment by boegel** [comment]
Going in, thanks @JackPerdue!

**Pull request merged**